helix-term/src/commands/jump/annotate.rs

```diff
@@ -0,0 +1,73 @@
+use super::JumpAnnotation;
+use crate::commands::Context;
+use helix_core::text_annotations::Overlay;
+use helix_view::input::KeyEvent;
+use std::rc::Rc;
+
+pub const JUMP_KEYS: &[u8] = b"etovxqpdygfblzhckisuran";
+
+pub fn apply_dimming(_ctx: &mut Context) {
+    // let (view, doc) = current!(ctx.editor);
```

Comment:
These unused comments are gone in the next commit. I was a bit lazy and didn't remove it here due to merge conflicts..